install_config/uninstalling-pipelines.adoc

```diff
@@ -8,21 +8,24 @@ toc::[]
 
 Cluster administrators can uninstall the {pipelines-title} Operator by performing the following steps:
 
-. Delete the Custom Resources (CRs) that were added by default when you installed the {pipelines-title} Operator.
-. Delete the CRs of the optional components such as {tekton-hub} that depend on the Operator.
+. Delete the `TektonConfig` Custom Resource (CR), as well as the CRs for the optional components, `TektonHub` and `TektonChains`, if these CRs exist.
 +
 [CAUTION]
 ====
-If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
+If you uninstall the Operator without removing the CRs of optional components, you cannot remove the components later.
 ====
 . Uninstall the {pipelines-title} Operator.
+. Delete the Custom Resource Definitions (CRDs) of the `operator.tekton.dev` group.
```

Comment:
While it's necessary to remove optional components' CRs, do we also need to remove CRDs? 😕

Reply:
@ppitonak this is what @piyush-garg told me on Slack. @piyush-garg can you confirm?